File: MantikIslemleri.py

sayi0 = int(input("Birinci sayıyı giriniz: "))#1.sayıyı kullanıcıdan integer olarak istedik
sayi1 = int(input("İkinci sayıyı giriniz: ")) #2.sayıyı kullanıcıdan int olarak istedik

if sayi0 == sayi1:  #Birinci sayi ile İkinci sayının eşitliğini kontrol ediyoruz
    print("\n Sayılar birbirlerine eşittir.\n Sayılar farklı değildir.\n Birinci sayı küçük değildir.") # ters "slash n" bir alt satıra geçmesini sağlar
    print(" İkinci sayı küçük değildir. \n Birinci sayı küçük eşittir.\n İkinci sayi küçük eşittir")

elif sayi0 != sayi1: #Birinci sayi ile İkinci sayının eşitsizliğini kontrol ediyoruz
    print("\n Sayılar eşit değildir.\n Sayılar farklıdır.")
    if sayi0 < sayi1: #Eğer birinci sayı ikinci sayıdan küçük ise bu işlemleri yap
        print(" Birinci sayi küçüktür.\n İkinci sayı küçük değildir.\n Birinci sayı küçük eşittir.")
    elif sayi0 > sayi1: #Eğer ikinci sayı birinci sayıdan küçük ise bu işlemleri yap
        print(" irinci sayı büyüktür.\n İkinci sayı büyük değildir.\n Birinci sayi büyük eşittir.")

File: MantikIslemleri.js
let number1 = prompt("Birinci sayiyi giriniz: ","5")
let number2 = prompt("İkinci sayiyi giriniz: ","10")

n1 = parseInt(number1)
n2 = parseInt(number2)

if(n1 == n2){
    console.log("Sayılar eşittir")
    console.log("Sayılar farklı değildir")
}
if(n1 != n2){
    console.log("Sayılar eşit değildir")
    console.log("Sayılar farklıdır")
}
if(n1<n2){
    console.log("İlk sayı küçüktür")
    console.log("İkinci sayı küçük değildir")
}
if(n1>n2){
    console.log("İkinci sayı büyüktür")
    console.log("İlk sayı küçük değildir")
}
if(n1<=n2){
    console.log("İlk sayı küçük eşittir")
}
if(n1>=n2){
    console.log("İkinci sayı küçük eşittir")
}
